backend/qemu.pm

```diff
@@ -545,16 +545,18 @@ sub start_qemu {
                 # XXX: workaround for OVMF wanting to write NVvars into first FAT partition
                 # we need to replace -bios with proper pflash drive specification
                 $params[-1] .= ',snapshot=on' if $vars->{UEFI};
-                push(@params, '-device', "$vars->{CDMODEL},drive=cd0,bus=scsi0.0");
+                my $bus = $vars->{CDMODEL} ne 'ide-cd' ? ',bus=scsi0.0' : '';
```

Comment:
I could imagine how to do it if the push(@params, *) would be same in both cases
Now I have no idea what you mean